Applications/Games/Space Engineers/Steam/script.js

```diff
@@ -0,0 +1,27 @@
+include("engines.wine.quick_script.steam_script");
+include("engines.wine.verbs.dotnet472");
+include("engines.wine.verbs.vcrun2017");
+include("engines.wine.verbs.dxvk");
+include("engines.wine.verbs.faudio");
+include("engines.wine.plugins.override_dll");
+
+new SteamScript()
+    .name("Space Engineers")
+    .editor("Keen Software House")
+    .author("Zemogiter")
+    .appId(244850)
+    .wineVersion("4.11")
```

Comment:
Why not `LATEST_DEVELOPMENT_VERSION`?

Reply:
Same reason as Subnautica before it was stable enough. Basically somewhere between wine 4.0.1 and wine 4.9 the game crashes even with all the fixes applied. This could happen again with newer versions being set as `LATEST_DEVELOPMENT_VERSION` in case they introduce some regression. I know we should minimize using specific wine version because user's disk space - you told me that already. But I think that making sure the software works is more important. I did through testing on 4.11

Reply:
The problem is: Wine is not the only thing that changes. For example, it can very well happen that Steam is updated and does not work on an old Wine version anymore.